src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -1032,7 +1033,8 @@ public SocketError ReceiveFromAsync(IList<ArraySegment<byte>> buffers, SocketFla
             }
         }
 
-        public SocketError ReceiveMessageFrom(byte[] buffer, int offset, int count, ref SocketFlags flags, byte[] socketAddress, ref int socketAddressLen, bool isIPv4, bool isIPv6, int timeout, out IPPacketInformation ipPacketInformation, out int bytesReceived)
+        public SocketError ReceiveMessageFrom(
```

Comment:
I started doing that, but it added a huge amount of duplicated code, for basically no reason other than consistency.